doc/source/ray-core/actors/fault-tolerance.rst

```diff
@@ -99,13 +99,13 @@ You can experiment with this behavior by running the following code.
         print(counter)  # Prints the sequence 1-10 5 times.
 
     # After the actor has been restarted 5 times, all subsequent methods will
-    # raise a `RayActorError`.
-    for _ in range(10):
+    # raise a `RayActorError` if the actor dies again.
+    for _ in range(11):
```

Comment:
To better match the comment, I think we should do a for loop of 60 times before and here just call it once.